### ISSUE TITLE
error TS7006: Parameter 'T' implicitly has an 'any' type.

### ISSUE THREAD

**Issue by lancon13** [jriecken/dependency-graph#19]
Hi,

I have the typescript compiler reported /lib/index.d.ts(33,31): error TS7006: Parameter 'T' implicitly has an 'any' type.

Looks like there is typo under the file lib/index.d.ts

```
/**
     * set the data for an existing node (will throw an Error if the node does not exist)
     * @param {string} name
     * @param data
     */
    setNodeData(name: string, T): void;
```

Should be like below?

```
/**
     * set the data for an existing node (will throw an Error if the node does not exist)
     * @param {string} name
     * @param data
     */
    setNodeData(name: string, data?: T): void;
```

I am new to typescript, maybe I haven't got my tsconfig.json setup properly..

**Comment by jriecken** [comment]
You are correct! I've just pushed a fix (and released version 0.5.2)

**Comment by lancon13** [comment]
It works fine now. Thanks and like your work!
